Game/SpriteHandlers/SpriteFactory/ParticleSpriteFactory.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using MainGame.SpriteHandlers.ItemSprites;
+
+namespace MainGame.SpriteHandlers
+{
+    public enum ParticleSpriteTypes
+    {
+
+    }
+
+    public static partial class SpriteFactory
+    {
+
+
+        public static ISprite CreateDeathParticles()
+        {
+
+
+            return new ParticleSprites.DeathParticles(
+                TextureMap["ParticleSprites"], SpriteBatch,
+                numRows: 1,
+                numColumns: 2,
+                numberOfFrames: 2,
+                frameHeight: 16,
```

Comment:
frameWidth and frameHeight already have default values of 16. I don't think this is necessary. This applies to the rest of the factory too.